dev/test-studio/plugins/locale-no-nb/bundles/studio.ts

```diff
@@ -207,114 +421,330 @@ const studioResources: Record<StudioLocaleResourceKeys, string> = {
+
+  /** Title for error when a filter cannot be displayed (mainly a developer-oriented error) */
+  'search.error.display-filter-title': 'En feil oppstod under visning av dette filteret.',
+
+  /** Description for error when a filter cannot be displayed, describes that you should check the schema */
+  'search.error.display-filter-description':
+    'Dette kan indikere ugyldige alternativer definert i skjemaet ditt.',
+
+  /** Label for action to clear recent searches */
+  'search.action.clear-recent-searches': 'Fjern nylige søk',
+
+  /** Dialog title for action to select an asset of unknown type */
+  'search.action.select-asset': 'Velg element',
+
+  /** Dialog title for action to select an image asset */
+  'search.action.select-asset_image': 'Velg bilde',
+
+  /** Dialog title for action to select a file asset */
+  'search.action.select-asset_file': 'Velg fil',
+
+  /** Text displayed when either no document type(s) have been selected, or we need a fallback, */
+  'search.action.search-all-types': 'Søk i alle dokumenter',
+
+  /** Text displayed when we are able to determine one or more document types that will be used for searching */
+  'search.action.search-specific-types': `Søk etter {{types, list}}`,
+
+  /** Text displayed when we are able to determine one or more document types that will be used for searching, but cannot list them all within the space assigned by the design */
+  'search.action.search-specific-types-truncated': `Søk etter {{types, list}} +{{count}} flere`,
+
+  /** In the context of a list of document types - no filtering selection has been done, thus the default is "all types". */
+  'search.document-type-list-all-types': 'Alle typer',
+
+  /** A list of provided types, formatted with the locales list formatter. */
+  'search.document-type-list': `{{types, list}}`,
+
+  /** A list of provided types that has been truncated - more types are included but not displayed, thus we need to indicate that there are more. */
+  'search.document-type-list-truncated': `{{types, list}} +{{count}} flere`,
+
+  /** Accessibility label for when the search is full screen (on narrow screens) and you want to close the search */
+  'search.action.close-search-aria-label': 'Lukk søk',
+
+  /** Accessibility label for when the search is full screen (on narrow screens) and you want to toggle filters */
+  'search.action.toggle-filters-aria-label_hide': 'Skjul filtre',
+  'search.action.toggle-filters-aria-label_show': 'Vis filtre',
+
+  /** Label for instructions on how to use the search - displayed when no recent searches are available */
+  'search.instructions': 'Bruk <ControlsIcon/> for å rafinere søket.',
+
+  /** --- Help & Resources Menu --- */
+
+  /** Title for help and resources menus */
+  'helpResources.title': 'Hjelp og ressurser',
+
+  /** Information for what studio version the current studio is running */
+  'helpResources.studio-version': `Sanity Studio versjon {{studioVersion}}`,
```

Comment:
camelCase spotted (there’s more of the same further down)

Reply:
Thanks for spotting!